components/admin_console/billing/billing_subscriptions.scss

```diff
@@ -196,3 +196,57 @@
         text-decoration: none;
     }
 }
+
+.customerCancelSubscriptionSection {
```

Comment:
nit: I would drop the word `customer` on all these names so they are not so wordy

```suggestion
.cancelSubscriptionSection {
```

Reply:
agree. Done!